src/util/bootstrap.helpers.js

```diff
@@ -57,7 +57,7 @@ export function bootstrapAxios() {
       if (get(error, 'response.status') === 403) {
         localStorage.removeItem('sessionId');
         localStorage.removeItem('appState');
-        history.push('/get-started?logout=true');
+        history.push('/map?logout=true');
```

Comment:
we should probably replace this whole block with a logrocket breadcrumb, not just change this redirect.